docker/Dockerfile

```diff
@@ -22,13 +22,16 @@ fi
 rm -rf /var/lib/apt/lists/*
 
 useradd -ms /bin/bash -d /home/orquestra orquestra --uid 1000 --gid 100
+
+mkdir -p /opt/orquestra
+chown -R 1000:100 /opt/orquestra
 EOF
 
 USER 1000
 WORKDIR /home/orquestra
 
-ENV VIRTUAL_ENV=/home/orquestra/venv
-RUN python -m venv "$VIRTUAL_ENV"
+ENV VIRTUAL_ENV=/opt/orquestra/venv
+RUN python -m venv --system-site-packages "$VIRTUAL_ENV"
```

Comment:
**question**: I don't usually use the `--system-site-packages` option, which makes my venvs fully isolated apart from Python's standard library. What's the backstory for this change?

Reply:
If users want to install packages in their home folder (the home folder is persisted), the venv needs access to the user site packages.
There's no option to use only the user site packages.
I'm not sure we've done enough investigating the other consequences to make this change.
For example, can I uninstall packages OK, will the system site packages cause an issue, etc.